qmetaobject_impl/src/lib.rs

```diff
@@ -89,7 +89,7 @@ pub fn qenum_impl6(input: TokenStream) -> TokenStream {
     qobject_impl::generate_enum(input, 6)
 }
 
-/// Implementation of the qmetaobject::qrc! macro
+// Implementation of the qmetaobject::qrc! macro
```

Comment:
why remove this documentation?

Reply:
Because it showed up at the bottom of generated docs. And that wasn't pretty.

Reply:
It stacks up with documentation on the re-export in the user-facing `qmetaobject` crate.

Reply:
For the reference, here it is at the bottom: https://docs.rs/qmetaobject/0.2.2/qmetaobject/macro.qrc.html

Reply:
ah ok.